src/System.Diagnostics.Process/src/PinvokeAnalyzerExceptionList.analyzerdata.uap

```diff
@@ -2,4 +2,3 @@ advapi32.dll!OpenProcessToken
 kernel32.dll!GetProcessAffinityMask
 kernel32.dll!SetProcessAffinityMask
 kernel32.dll!SetThreadAffinityMask
-ntdll.dll!NtQuerySystemInformation
```

Comment:
that is correct, sorry. Opened the PR in CodeFlow and for some reason it showed this line as if it was added, instead of removed. 😄